source/cms.c

```diff
@@ -20,6 +20,17 @@
 #define ENVELOPED_DATA_VERSION (2)
 #define ENVELOPED_DATA_RECIPIENT_VERSION (2)
 
+static inline int get_any_ber_asn1_element(
+    CBS *cbs,
+    CBS *out,
+    CBS_ASN1_TAG *out_tag,
+    size_t *out_header_len) {
+
+    int indefinite_temp;
```

Comment:
Why do we introduce this indirection? Is this not the same as calling it without, as we were doing before?